pkg/prom/model.go

```diff
@@ -38,6 +39,15 @@ var C = Costs{
 		},
 		[]string{"account"},
 	),
+	SpotRequest: *prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "rebuy",
+			Subsystem: "cost_exporter",
```

Comment:
We might have to change this?

Reply:
that would be a bigger thing, i guess, with renaming the repo and so on. 
i would do that next friday.